documentation/examples/cqrs/web-client/src/main/scala/cqrs/webclient/Main.scala

```diff
@@ -74,53 +71,61 @@ object Main {
       }
     }
 
-    val app =
-      <article>
-        <h1>Meters</h1>
-        <p>
-          <input type="text" placeholder="New meter name" id={newMeterId} required="required" />
-          <button onclick={onNewMeterClicked}>Create</button>
-        </p>
-        {
-        metersVar.map { meters =>
-          if (meters.isEmpty) {
-            <p>No meters yet!</p>
-          } else {
-            <div>
-                {
-              meters.toSeq.map {
-                case (_, meter) =>
-                  <section>
-                      <h2>{meter.label}</h2>
-                      <p>
-                        <table>
-                          <thead>
-                            <th>Time</th><th>Value</th>
-                          </thead>
-                          <tbody>
-                            {
-                    meter.timeSeries.toSeq.map {
-                      case (instant, value) =>
-                        <tr><td>{instant.toString()}</td><td>{value.toString()}</td></tr>
-                    }
-                  }
-                          </tbody>
-                        </table>
-                        <input placeholder="New value" required="required" id={
-                    s"value-${meter.id.toString}"
-                  } />
-                        <button onclick={onAddValueClicked(meter)}>Add</button>
-                      </p>
-                    </section>
-              }
+    val app = article(
+      h1("Meters"),
+      p(
+        input(
+          `type` := "text",
+          placeholder := "New meter name",
+          idAttr := newMeterId,
+          required := true
+        ),
+        button(onClick.map(_ => ()) --> onNewMeterClicked, "Create")
+      ),
+      child <-- metersVar.signal.map { meters =>
+        if (meters.isEmpty) {
+          p("No meters yet!")
+        } else {
+          div(
+            meters.toSeq.map {
```

Comment:
Keep in mind that metersVar contains a Map, which is by its nature unsorted, so it will be rendered in arbitrary order determined by toSeq. You could make metersVar contain a List[Meter] instead, or define some sorting order if you wanted to address that.